events-manager.js

```diff
@@ -15,14 +15,22 @@ CTLEventsManager.filteredEvents = [];
  * instance for each one. This also indexes these items with the
  * given search index.
  *
- * Returns the array of CTLEvents.
+ * @param eventsJson = the JSON object in the format provided by
+ *                     Bedeworks
+ *
+ * @param filterAfterDate = a Date object to filter events by. This will
```

Comment:
I would change this param name to `startDate`, or something else a little clearer. When I see `filterAfterDate` it's not obvious to me whether you're including or excluding objects after this date.